pytest.ini

```diff
@@ -1,2 +1,2 @@
 [pytest]
-testpaths = db webserver hl_extractor dataset_eval
```

Comment:
Make sure you don't accidentaly commit changes like this. This will break our tests and stop them working for other developers. Instead, you can give a specific path to test on the command line when you want to check your tests:

    py.test webserver/views/api/v1